tinytuya/wizard.py

```diff
@@ -158,15 +158,22 @@ def wizard(color=True, retries=None, forcescan=False, nocloud=False):
         tuyadevices = cloud.filter_devices( json_data['result'] )
 
     for dev in tuyadevices:
```

Comment:
Can we add a comment here to explain what this section does?

Reply:
Actually debugging the conditionals is a bit difficult. I suggest we add a comments to explain what the loop and conditional branches are telling us.  

Is it this?  Scan through all devices.  For each device,  check to see if its key matches another device (that isn't itself and doesn't have a 'sub' key) and if found, make that the parent of this device.